src/views/Nft/market/Collection/IndividualNFTPage/DetailsCard.tsx

```diff
@@ -31,14 +31,16 @@ const DetailsCard: React.FC<DetailsCardProps> = ({ contractAddress, ipfsJson })
           <LongTextContainer bold>{contractAddress}</LongTextContainer>
         </Link>
       </Flex>
-      <Flex justifyContent="space-between" alignItems="center">
-        <Text fontSize="12px" color="textSubtle" bold textTransform="uppercase">
-          IPFS JSON
-        </Text>
-        <Link external href={ipfsJson}>
-          <LongTextContainer bold>{ipfsJson}</LongTextContainer>
-        </Link>
-      </Flex>
+      {ipfsJson && (
```

Comment:
API does not return ipfsJson link, and I'm kinda hesitant to make subgraph request purely to get this url... might come back to it in future PR